index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>portfolio-thais</title>
+</head>
+<body>
+    <header>
+        <img src=""> <!--link to image for log + ned to add image to project??-->
+        <nav></nav>
+    </header>
+    <a> href=""></a> <!--arborescence cliquable???? link to what?-->
+    <main>
+        <section>
+            <img src=""> <!--link photo-->
+            <aside>
+                <h1></h1><!--titre-->
+                <p></p> <!--description-->
+            </aside>
+            <article></article> <!--what ellipsis is?-->
+            <button></button> <!-- is there a hyperlink for the button to access article?-->
+        </section>
+        <aside>
+            <p>thais information</p>
+        </aside>
+        <section>
+            <img src=""> <!--add slide auto on img-->
+        </section>
+        <section>
+            <div>
+                <article>Article A</article>
+            </div>
+            <div>
+                <article>Article B</article>
+            </div>
+            <div>
+                <article>Article C</article>
+            </div>
+            <div>
+                <article>Article D</article>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <ul>
```

Comment:
list is great but your implementation is very bad

Reply:
"<li></li>" corrigées